### DISCUSSION TITLE
change config for sanity.cli.ts command line

### DISCUSSION THREAD

**Discussion by safejace** [sanity-io/sanity#3837] [Sanity Studio Support]
with sanity v2 i was able to create `.env.{something}` and be able to run commands for specific environments

`$ SANITY_ACTIVE_ENV=something sanity exec ...`

with sanity v3 the project id and dataset are predefined and there are no docs about how to set the dataset in the command line

**Comment by englut** [comment]
If you use `dotenv` and `process.env.SANITY_STUDIO_PROJECT_ID` and `process.env.SANITY_STUDIO_DATASET`, you can re-use the same `.env` for both configs.
